### PR TITLE
[OYPD-466] YouTube icon in the footer is not announced needs alt tesxt added.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Issue: https://propeople-us.atlassian.net/browse/OYPD-466
Drupal.org issue: https://www.drupal.org/node/2884135

Steps for review:
- [x] go to any page
- [x] scroll down and find the footer social icons
- [x] verify youtube icon looks as a link

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1351 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1351 |
| Installation process | http://install.openy-dev.ffwua.com/build1351/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1351/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1351/profiles/contrib/openy/build/reports/behat

File: modules/openy_features/openy_demo_content/modules/openy_demo_bfooter/config/install/migrate_plus.migration.block_content_footer.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ source:
         <ul class="list-inline">
         	<li><a class="fa fa-facebook" href="#">Facebook</a></li>
         	<li><a class="fa fa-twitter" href="#">Twitter</a></li>
-        	<li><a class="fa fa-youtube href="#">Youtube</a></li>
+        	<li><a class="fa fa-youtube" href="#">Youtube</a></li>
```
Comment (alexschedrov) [inline modules/openy_features/openy_demo_content/modules/openy_demo_bfooter/config/install/migrate_plus.migration.block_content_footer.yml]: This is good catch, however we still have to add `title=""` for all those links.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1364 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1364 |
| Installation process | http://install.openy-dev.ffwua.com/build1364/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1364/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1364/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Review by alexschedrov — APPROVED** [review]

**Pull request merged**